### DISCUSSION TITLE
Any examples as an image picker?

### DISCUSSION THREAD

**Discussion by Fjacquette** [igordanchenko/react-photo-album#66] [Q&A] [answered]
First off, thanks for putting this together; I found the project when I started bumping into some of the challenges around react-photo-gallery getting dusty.

Are you aware of anyone having put together an example using this as an image picker?

**Comment by igordanchenko** [comment, marked as answer]
Hi there!

Unfortunately, I'm not aware of a readily available demo, but you may want to take a look at [GoogleForCreators
/ web-stories-wp](https://github.com/GoogleForCreators/web-stories-wp) or [sanity-plugin-asset-source-unsplash](https://github.com/sanity-io/sanity-plugin-asset-source-unsplash). Both projects are using react-photo-album as media library picker.
